### ISSUE TITLE
[Mainissue] Mining & Herbalism Issues

### ISSUE THREAD

**Issue by Lonerin** [dalaranwow/dalaran-wow#1208]
I'll make it short. I found out that copper deposits, and arguably any mining deposits drop way too high amount of ores, like x7-x8-x9 with an insane amount of deposits available in a short range area.
The opposite happens for herbalism, the drop rates are fine, 1-3, but there are way too few herbs around. I found this on Kirin Tor realm aswell, expecially in Northrend.
I'd like other points of view on this subject.
Thank you

todo:
- [ ] 1. Remove duplicate spawnpoints for Mining & Herbalism
- [ ] 2. Add missing spawnpoints for Mining & Herbalism
- [ ] 3. Add better pooling system for Mining & Herbalism

**Comment by Rushor** [comment]
DB/GameObject: Implement Herb Spawn Pool System for Alterac Mountains

* updates https://github.com/dalaranwow/dalaran-wow/issues/1208
* commit: 3073e015fd8ed4a52e288658a1c3d10ca5945a7a

**Comment by Rushor** [comment]
DB/GameObject: Implement Herb Spawn Pool System for Arathi Highlands

* updates https://github.com/dalaranwow/dalaran-wow/issues/1208
* commit: 7208c78455599896571f7fad95a6197fa872bf25

**Comment by Rushor** [comment]
DB/GameObject: Implement Herb Spawn Pool System for Isle of Quel'Danas

* updates https://github.com/dalaranwow/dalaran-wow/issues/1208
* commit: 691c7a2c69d57754e00f0c8a5d7603afbdf7072b

Herb pooling for the three zones i posted above should be fine now. i will close the issue, so the testers know they need to test it. reopen it once the herb fixes are on live. i will fix the mine poolimg later.

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by Rushor** [comment]
https://github.com/dalaranwow/dalaran-wow/issues/4334

Couple Issues
1.Can the respawn rate on actual Frost Lotus be check in Wintergrasp or the spawn share rate? There should be 157 spawn locations.

2. Herb phasing in wintergrasp
**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
**Issue 1** 
I have only ever found 1 Frost Lotus spawn in Wintergrasp. Frost Lotus should share respawns with frozen ferbs. 
Comment from wowhead patch 3.0.8
I been running around Wintergrasp and marking out where I found Frost Lotuses and I have noticed that they share the same spawn spot as Frozen Herbs.
**Issue 2**
I have noticed a herb and some ore mines phased based on where you are in witergrasp. 
Some are based on a Dragon Blight phase. Up at glacial falls.
**Expected behaviour**: Tell us what should happen instead
Frost Lotus should share respawn with frozen herbs.
No herbs should phase in Wintergrasp.
**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**
Herb can be seen
![wowscrnshot_012617_222348](https://cloud.githubusercontent.com/assets/12252441/22359577/7243b124-e416-11e6-814f-daf7d8637f12.jpg)

Move a inch herb dissappears.
![wowscrnshot_012617_222353](https://cloud.githubusercontent.com/assets/12252441/22359582/7ca791d0-e416-11e6-8984-dece0d2c8436.jpg)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
[Frozen Herb](http://www.wowhead.com/object=190176/frost-lotus#herbalism)
You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by srcssrcs** [comment]
so you guys gonna fix your broken server or what?

**Comment by srcssrcs** [comment]
LMAO nice fix, retard
